tools/i18n/regenerate_en_template.py

```
@@ -0,0 +1,199 @@
+            js.update(index_js)
+            save_json(en_template_dir / 'js.json', js)
+            print(f"✓ Merged: {len(js)} total js strings")
+        except (json.JSONDecodeError, IOError) as e:
+            print(f"Warning: Failed to merge index.js: {e}")
+    
+    # Step 7: Create metadata
+    print("Step 7: Creating metadata...")
+    commit_hash = ''
+    commit_msg = ''
+    for ref in ['upstream/main', 'upstream/master', 'origin/main', 'origin/master']:
+        result = subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)